name/utils.py

```diff
@@ -62,9 +62,12 @@ def resolve_type(q_type):
 
     q_type = q_type.split(',')
 
-    # Iterate through the Name Types and create a list of Name IDs
-    # using the corresponding strings values in q_type
-    return [k for k, v in Name.NAME_TYPE_CHOICES if v in q_type]
+    if "Any Type" in q_type:
+        return range(5)
```

Comment:
I now remember why I did that initially. I had at first actually added another value to NAME_TYPE_CHOICES before deciding that wasn't the best way to do things. Anyways, fixing.